app/controllers/thredded/autocomplete_users_controller.rb

```diff
@@ -30,6 +30,7 @@ def users_by_prefix
       if query.length >= Thredded.autocomplete_min_length
         DbTextSearch::CaseInsensitive.new(users_scope, Thredded.user_name_column).prefix(query)
           .where.not(id: thredded_current_user.id)
+          .order(Thredded.user_class.arel_table[Thredded.user_name_column].lower.asc)
```

Comment:
We'll get better results if we use the appropriate case-insensitive method for each database, especially for things like the German "ß", the Turkish "ı", etc.

Let's add a method to `db_text_search` instead?
Using it could look something like this:

```ruby
case_insensitive = DbTextSearch::CaseInsensitive.new(users_scope, Thredded.user_name_column)
case_insensitive.prefix(query)
  .where.not(id: thredded_current_user.id)
  .order(case_insensitive.column_for_order(:asc))
```

Reply:
https://github.com/thredded/db_text_search/pull/5